lib/handlers/injector-sor.ts

```diff
@@ -262,13 +271,23 @@ export abstract class InjectorSOR<Router, QueryParams> extends Injector<
           process.env.TENDERLY_ACCESS_KEY!,
           v2PoolProvider,
           v3PoolProvider,
-          provider,
+          provider as ethers.providers.JsonRpcProvider,
```

Comment:
any reason why this is not `jsonRpcProvider`?

Reply:
I addressed it